openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/SWIGTrainResources.java

```diff
@@ -34,7 +34,7 @@
  * @author Alberto Ferreira (alberto.ferreira@feedzai.com)
  * @since 1.0.10
  */
-public class SWIGTrainResources implements AutoCloseable {
+class SWIGTrainResources implements AutoCloseable {
```

Comment:
The last comment, you made class package-private you should have also made constructor package-private.